public/v2/apps/spigot.json

```diff
@@ -0,0 +1,74 @@
+{
+   "captainVersion": "2",
+   "documentation": "https://github.com/nimmis/docker-spigot",
+   "displayName": "Spigot",
+   "description": "Spigot is a performance optimized server for Minecraft with plugin support.",
+   "dockerCompose": {
+      "services": {
+         "$$cap_appname": {
+            "notExposeAsWebApp": "true",
+            "ports": [
+               "$$cap_port:25565"
+            ],
+            "environment": {
+               "EULA": "$$cap_eula",
+               "SPIGOT_VER": "$$cap_version",
+               "MC_MAXMEM": "$$cap_maxmem",
+               "MC_MINMEM": "$$cap_minmem",
+               "SPIGOT_AUTORESTART": "$$cap_autorestart"
+            },
+            "image": "nimmis/spigot:latest",
+            "restart": "always",
+            "volumes": [
+               "$$cap_appname-data:/minecraft"
+            ]
+         }
+      },
+      "version": "2",
+      "volumes": {
+         "$$cap_appname-data": {}
+      }
+   },
+   "instructions": {
+      "end": "The container will now download and build the Spigot Minecraft server, so you will need a few minutes before you can access it. Check the status in the logs.",
+      "start": "You have to agree to the Minecraft EULA to be able to use this software: https://account.mojang.com/documents/minecraft_eula"
+   },
+   "variables": [
+      {
+         "label": "Game Server Port (TCP/UDP)",
+         "description": "This is the port the server will expose for players to join.",
+         "defaultValue": "25565",
+         "id": "$$cap_port"
+      },
+      {
+         "label": "EULA",
+         "description": "IMPORTANT: Read the EULA at https://account.mojang.com/documents/minecraft_eula and set this field to true if you agree. This is a prerequisite to use this software.",
+         "defaultValue": "",
+         "id": "$$cap_eula"
+      },
+      {
+         "label": "Spigot Version",
+         "description": "Versions as available on https://www.spigotmc.org/wiki/buildtools/#versions (e.g. 1.15.2 or latest)",
+         "defaultValue": "latest",
```

Comment:
You can freeze the version by using sha tag:
```
sha256:b968685fadb79e6dacca8efc030ee53f695441b8d6dfde4b86e027ba1f6796db
```

https://hub.docker.com/layers/nimmis/spigot/latest/images/sha256-b968685fadb79e6dacca8efc030ee53f695441b8d6dfde4b86e027ba1f6796db?context=explore

Reply:
Thanks, I added it as the default version variable!